dapps/marketplace/src/index.js

```diff
@@ -2,7 +2,15 @@
 const appHash = process.env.GIT_COMMIT_HASH || 'marketplace'
 const ognNetwork = localStorage.ognNetwork
 if (localStorage.appHash !== appHash) {
+  let exceptions = ['growth_auth_token']
```

Comment:
Awesome. Thanks for doing that. Looks great.
One minor nit pick: add a comment to explain why we make an exception for growth token.
For dummies like me that won't remember when reading this code a year from now... :)